type-generation/README.md

```diff
@@ -0,0 +1,66 @@
+# GraphApi.io Type generation
+
+In this simple example we will show how to quickly generate typescript definitions for you Graph Api.
+
+## Steps
+
+### Clone this repository
+
+```
+git clone https://github.com/graphapi-io/resources.git
+cd resources
+```
+
+### Install dependencies
+
+```
+ npm i
+```
+
+### Get your api key
+
+1.  Go into your API Settings on https://my.graphapi.io
+1.  You may be asked to log in to your API. If you haven't done it yet, sign up using the same email that you use for signing into the my.graphapi.io
+1.  Find "Authorization Settings"
+1.  Under "BASIC AUTHORIZATION" you should see "DEFAULT KEY". Technically you could use it, but it's a best practice to generate a separate key for each application/use case.
+1.  Click "Add Api Key" button in the section below
+    ![Api Keys Section](./docs/img/add-api-key-1.png?raw=true)
+1.  Provide a meaningful name and click "Create"
+    ![Api Keys Section](./docs/img/add-api-key-2.png?raw=true)
+1.  Copy the key
+    ![Api Keys Section](./docs/img/add-api-key-3.png?raw=true)
+1.  In the repository copy the .env.example file into .env file
```

Comment:
better: copy the .env.example file to .env 

We could also just do prompts here:
`$ cp .env.example .env`